### PR TITLE
License review per #1352

### PR DESCRIPTION
**Pull request by robocoder**
- add LEGALNOTICE.md
- add commons-io-2.2.jar
- remove license/LICENSE.junit.txt (see zaproxy-test repo)
- remove JBroFuzz*.jar (see zap-extensions repo)

### CODE REVIEW

**Comment by thc202** [comment]
The reference to zaproxy-test repo should be removed, the "license" directory is for licenses of Java ZAP API client (it's OK to remove it since the library is no longer bundled in the jar).

Why not Apache Commons IO 2.4? or why 2.2? (the bundled version is/was 1.4)

This change breaks the compatibility with 2.4.0.

**Comment by robocoder** [comment]
According to the release notes, 2.2 is source and binary compatible with 1.4 and 2.1.

The 2.4 release notes mentioned source incompatibility.

**Comment by robocoder** [comment]
I'll amend the comment in the morning.

**Comment by thc202** [comment]
Version 2.4 is also binary compatible (moreover most of the add-ons are already being compiled using 2.4 so any compilation error would have been fixed by now).

**Comment by robocoder** [comment]
Ok.  Done.

File: LEGALNOTICE.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+COPYRIGHT
+---------
+
+OWASP Zed Attack Proxy (ZAP)
+
+The software package is:
+
+    Copyright (C) 2010  Simon Bennetts
```
Comment (psiinon) [inline LEGALNOTICE.md]: Could this be changed to something like "Copyright (C) 2010-2015 The ZAP Development Team" ?
I realize that the "ZAP Development Team" might be less well defined, but I dont think I should be taking credit for all the work other people contribute :)

Comment (robocoder) [inline LEGALNOTICE.md]: I'll make that change but IANAL, so I can't advise you of the implications of not declaring a legal entity as the copyright holder. Perhaps something to reconsider if/when the team ever decides to seek trademarks.

Comment (psiinon) [inline LEGALNOTICE.md]: Or "Simon Bennetts and the ZAP Development Team" ??
IANAL either ;)

Comment (robocoder) [inline LEGALNOTICE.md]: Oops. I forgot to include a mention of Paros and the Clarified Artistic license. Any thoughts on the wording?

Comment (psiinon) [inline LEGALNOTICE.md]: Oh yeah - definitely need that!
Can you rip off some of the statements used here: https://github.com/zaproxy/zap-core-help/wiki/HelpParos ?

Comment (thc202) [inline LEGALNOTICE.md]: Worth checking too https://github.com/zaproxy/zaproxy/blob/develop/src/org/zaproxy/zap/view/AboutPanel.java#L54

Comment (robocoder) [inline LEGALNOTICE.md]: Done. Commits squashed.

**Comment by kingthorin** [comment]
Looks good to me

**Comment by kingthorin** [comment]
@robocoder Looks good to me, just needs a rebase

File: LEGALNOTICE.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+COPYRIGHT
+---------
+
+OWASP Zed Attack Proxy (ZAP)
+
+The software package is:
+
+    Copyright © 2010-2015 Simon Bennetts and the ZAP Development Team
```
Comment (psiinon) [inline LEGALNOTICE.md]: Should be 2016 now

**Comment by psiinon** [comment]
Looks good apart from the 2010-2015 in the LEGALNOTICE.md which is now out of date

**Pull request merged**